pkg/middlewares/redirect/redirect_scheme.go

```diff
@@ -14,10 +14,16 @@ import (
 
 const (
 	typeSchemeName  = "RedirectScheme"
-	uriPattern      = `^(https?:\/\/)?(\[[\w:.]+\]|[\w\._-]+)?(:\d+)?(.*)$`
+	uriPattern      = `^(https?:\/\/)(\[[\w:.]+\]|[\w\._-]+)?(:\d+)?(.*)$`
```

Comment:
```suggestion
	uriPattern      = `^((?:wss?|https?):\/\/)(\[[\w:.]+\]|[\w\._-]+)?(:\d+)?(.*)$`
```

We should probably also support `ws` and `wss` scripts.

Reply:
no because we replace it by http/https